rules/false_positives/clickhouse.yara

```diff
@@ -0,0 +1,15 @@
+rule clickhouse_binary : override {
+  meta:
+    malware_PlugX_config = "high"
```

Comment:
This should be `ignore` or something equivalent, it's a complete false-positive, AFAIK.

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).